docs/kratos/self-service/flows/user-logout.mdx

```diff
@@ -33,85 +57,24 @@ selfservice:
         default_browser_return_url: http://test.ory.sh:4000/
 ```
 
-<CodeTabs items={browserFlow} />
+```mdx-code-block
+  </TabItem>
+</Tabs>
+```
 
 If an error occurs, the browser is redirected to the [Error UI](./user-facing-errors.md).
 
-## Self-service logout for client-side browser applications
-
-Similar to server-side browser applications, Ory Identities first generates a Logout URL for a given Ory Session Cookie. However,
-you can simply call the Logout URL using an AJAX request. Ory Identities returns a `204 No Content` response on success or an
-error otherwise.
+## Client-side browser applications
```

Comment:
Should we just call this single page apps?

Reply:
How about `## Single page application (SPAs)` so people can recognize the buzzword (SPA) as well?